### PR TITLE
build(deps):  remove native code + unused pods

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [PHIRE-139] 

### Description

Removes some old pods that are no longer necessary:
- AROAuth previously used for twitter auth, not used in a _long_ time 
- AFNetworking, we are only using some of the subspecs, removed the rest
- EDColor, was used for some convenience methods but we have an extension that does the same thing
- ARGenericTableViewController, was only used in old admin menu

Removes some old native code: 
- Old admin menu, this has some cool tools but hasn't been given attention in a long time and most are broken, getting rid of let's us get rid of a fair bit of code

Cleans up unused imports:
- Went through a bunch of files checking for unused imports, there were quite a few

### Follow-ups

- Remove more code!

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Remove unused code + native dependencies - Brian 

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[PHIRE-139]: https://artsyproduct.atlassian.net/browse/PHIRE-139?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by damassi** [comment]
Thank you for following up on this 👍

File: ios/Artsy/ArtsyUIColors/UIColor+ArtsyColors.h [hunk 1]
```diff
@@ -43,4 +43,7 @@
 /// <br/> (#0eda83)
 + (UIColor *)artsyGreenRegular;
 
+// Allow creating colors with a hex code
++ (UIColor *)ar_colorWithHex:(UInt32)hex;
```
Comment (brainbicycle) [inline ios/Artsy/ArtsyUIColors/UIColor+ArtsyColors.h:47]: exposing a convenience method we already had to get rid of EDColor which we were using for same purpose

File: ios/Artsy/App/ARAppDelegate.mm [hunk 2]
```diff
@@ -10,44 +8,25 @@
...
 #import "ARCocoaConstantsModule.h"
 
-#import "UIDevice-Hardware.h"
-#import "ArtsyEcho.h"
-
 #import <react-native-config/ReactNativeConfig.h>
-
 #import <ObjectiveSugar/ObjectiveSugar.h>
-#import <React/RCTDevSettings.h>
-#import <React/RCTBridge.h>
```
Comment (brainbicycle) [inline ios/Artsy/App/ARAppDelegate.mm:50]: most of the header file deletions / noise is just removing and check to see if it still compiles.

**Review by gkartalis — APPROVED** [review]
on fire! 🧹🧹🧹🧹

**Review by MounirDhahri — APPROVED** [review]
LGTM - thanks Brian

**Pull request merged**